posts/templates/post_detail.html

```diff
@@ -40,23 +39,30 @@ <h2 class="blog-post-title bg-warning px-3">{{ post.title }}</h2>
             </div>
         </div>
 
-
         <hr>
         <div class="d-flex">
-
-            <a class="btn btn-outline-success me-2" href="comment/">Add comment</a>
-            <a class="btn btn-outline-dark me-2" href="{% url 'post_share' pk=post.id %}">Share this post </a>
-
-            <button data-id="{{ post.id }}" class="btn btn-secondary like-button me-2">
-                Like <span class="badge bg-primary" id="total_likes_badge">{{ total_likes }}</span>
-
-            </button>
-
-
-            <a href="{{ post.link }}" target="_blank" class="btn btn-outline-primary"> Goto> </a>
+                    <form method="POST" action="{% url 'add_comment_to_post' pk=post.id%}">
+                        {% csrf_token %}
+                        <a class="btn btn-outline-success me-2" type="submit">Add comment</a>
+                    </form>
```

Comment:
@sehmuzc We need to make post or space actions as POST requests since it is the best practice. I will refactor the old ones. Can you also create new ones like this please?

Reply:
Ok Ceyda, these are coming from the previous semester project.